### DISCUSSION TITLE
Preview explanation for docs

### DISCUSSION THREAD

**Discussion by toniengelhardt** [chakra-ui/ark#2613] [General]
Could you add a small note or FAQ to the docs explaining what the "preview" badge for components means? It doesn't seem clear whether they can be used already in prod (with expectations of small bugs, light/heavy changes to the API, etc.), or definitely should not. Thanks 🙏🏽

**Comment by cschroeter** [comment]
@toniengelhardt 

The preview badge indicates that we reserve the right to make breaking changes to the component without issuing a major release for the entire framework. This means semantic versioning does not apply to these components.

However, these components are tested and generally stable. Most changes involve name adjustments, with major refactoring happening only rarely.

We aim to remove the preview badge once we are confident that the component functions as designed.

**Reply by toniengelhardt** [reply]
Great! Thanks for clarifying 🙏🏽

**Comment by segunadebayo** [comment]
Closing — the docs site has been redesigned since this was filed.
